src/highlight.js

```diff
@@ -115,6 +117,12 @@ function createLineElement({
     children.unshift(getInlineLineNumber(lineNumber, inlineLineNumberStyle));
   }
 
+  if (wrapLongLines & showLineNumbers) {
+    properties.style
+      ? (properties.style['display'] = 'flex')
+      : (properties.style = { display: 'flex' });
```

Comment:
I guess these (and also the other 2 occurrences down below) could also be written like this:

```js
properties.style = { ...properties.style, display: 'flex' }
```

Even if `properties.style` is `undefined`, this still works.

Reply:
Nice, that's much better